### PR TITLE
Quickicon a11y blocks

### PR DESCRIPTION
**Pull request by chmst**
For quickicons several issues and comments were open. 

### Summary of Changes

-> All Buttons now are list elements < li >
-> If two buttons belong to the same component (list view and edit view), they are in a list < ul >. The screenreader NVDA seems to read this correct. 

-> The text on Add-Buttons is now the verb "Add" instead of "New".
-> From now on, the number can be switched off and on for all icons (also cache and checkin).

Every help for further improving accessibility of quickicons is highly appreciated.

### CODE REVIEW

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 1]
```diff
@@ -44,56 +35,96 @@
...
       background-color: var(--atum-link-color);
-      text-align: center;
+     } 
     }
-
-    span:last-child {
-      margin-left: 2rem;
+  }
+  
+  // Singe icon
```
Comment (brianteeman) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]:
```suggestion
  // Single icon 
```

**Comment by brianteeman** [comment]
Will try to take a look tomorrow

**Comment by chmst** [comment]
thx

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 2]
```diff
@@ -44,56 +35,96 @@
...
+      border: 1px solid $gray-400;
+	  
+      a {
+        overflow: hidden;
+        display: block;
+        padding-left: 2.4rem;
+        color: var(--atum-text-dark);
+	  
+     &:hover {
+      @include hover-focus-active {
```
Comment (bembelimen) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]: This leads to a double ```:hover```

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 3]
```diff
@@ -229,16 +271,17 @@
...
       @media (max-width: 1180px) {
         width: calc(50% - 1rem);
         max-width: calc(50% - 1rem);
         flex: 1 1 calc(50% - 1rem);
       }
       @media (max-width: 860px) {
-        width: calc(100% - 1rem);
-        max-width: calc(100% - 1rem);
-        flex: 1 1 calc(100% - 1rem);
+        width: 100%;
```
Comment (brianteeman) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]: do you really have to set them all to 100% ??

I dont' know about flex but I was taught
if width > max-width; let the browser use max-width.
if width < max-width; let the browser use width.

File: administrator/templates/atum/scss/blocks/_quickicons.scss (hunk 2 above)
Comment (chmst) [reply administrator/templates/atum/scss/blocks/_quickicons.scss]: fixed

File: administrator/templates/atum/scss/blocks/_quickicons.scss (hunk 3 above)
Comment (chmst) [reply administrator/templates/atum/scss/blocks/_quickicons.scss]: Fixed. With is here 100% per default,

**Comment by chmst** [comment]
@brianteeman would be great if you could have a look here. 

You wrote that the code is completely broken, could you please define what is broken and what you think would be a solution?

**Comment by brianteeman** [comment]
I tried but got this error when running npm 

```
Error: Undefined variable: "$quickicon-color-hover".
        on line 60 of administrator/templates/atum/scss/blocks/_quickicons.scss, in mixin `@content`
        from line 35 of media/vendor/bootstrap/scss/mixins/_hover.scss, in mixin `hover-focus-active`
        from line 58 of administrator/templates/atum/scss/blocks/_quickicons.scss
        from line 34 of administrator/templates/atum/scss/template.scss
>>           color: $quickicon-color-hover;
   -----------------^
```

Looks like you deleted the variable from administrator/templates/atum/scss/_variables.scss

**Comment by chmst** [comment]
Did you run npm build:css?

**Comment by brianteeman** [comment]
That is the error from npm

The problem is this commit https://github.com/joomla/backend-template/commit/ef27ec65b39f499e6b79470c103fe0699a9dd72a where you removed the variable definition but did not remove the use of the variable

**Comment by chmst** [comment]
Sorry, I need new glasses.

**Comment by brianteeman** [comment]
The site module gives a11y errors

Issue: <ul> and <ol> must only directly contain <li>, <script> or <template> elements
  List element has direct children that are not allowed inside <li> elements

The other quikicon modules dont so I am going to assume thats because they dont have the extra links

**Comment by brianteeman** [comment]
The screenreader seems happy and it seems ok to me now with the new count strings. I definitely would default to not have the counts.

Really needs stefan to check it as he is much more experienced than me

**Comment by chmst** [comment]
The combination of links to both views of a component is an own list with two elements. 
Tabbing is possible, for sighted users everything is visible, the screenreader seem to read correctly. So I don't know where the issue could be.

Maybe it is as with images. Missing alt texts always cause a a11y issue, even if is a pure decorative image without any meaning. 

Do you know a better solution than we have now?

**Comment by brianteeman** [comment]
I dont know where the issue is either. The same problem exists with the third level of menus. Hopefully I can get some advice tonight

**Comment by brianteeman** [comment]
Found the markup error - will try to write a fix for that tomorrow. It shouldnt hold up the merge of this pr though

**Review by bembelimen — APPROVED** [review]

**Pull request merged**